torch/csrc/jit/mobile/runtime_compatibility.h

```diff
@@ -1,11 +1,20 @@
 #pragma once
 
 #include <memory>
+#include <unordered_map>
 
 namespace torch {
 namespace jit {
 
+static const int NO_SCHEMA = -1;
+
+struct OperatorInfo {
```

Comment:
This is the same as in 57501 I'll consolidate the two.

Reply:
Would it be consolidated in this PR?

Reply:
I'll rebase the model api to be dependent on this one and have the model method use the runtimes definition